hnn_core/network.py

```diff
@@ -1253,23 +1253,54 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
 
         self.connectivity.append(deepcopy(conn))
 
-    def clear_connectivity(self):
-        """Remove all connections defined in Network.connectivity
+    def clear_connectivity(self, src_types="all"):
+        """Remove connections with src_type in Network.connectivity.
+
+        Parameters
+        ----------
+        src_types : list | 'all' | 'drives' | 'local'
+            Source types of connections to be cleared
+            'all' - Clear all connections (Default)
+            'drives' - Clear connections originating from external drives
+            'local' - Clear connections within cells
```

Comment:
```suggestion
            'drives' - Clear connections from the external drives to the local network
            'local' - Clear connections between cells of the local network
```
How about `'external'` or `'exogenous'` instead of `'drives'`, and `'internal'` or `'endogenous'` instead of `'local'`? I've gotten the impression our documentation is already a little hazy on how we delineate between these two distinct categories of connections.

Reply:
@jasmainak @ntolley wdyt?

Reply:
I like 'external' and 'internal' ... very explicit, clear and I prefer it over exogenous/endogenous which is a bit too academic.

Reply:
That's fair!

Reply:
Sure will rename accordingly